src/app/backing/components/BuilderCard/BuilderCard.tsx

```diff
@@ -55,47 +70,60 @@ export const BuilderCard: FC<BuilderCardProps> = ({
       />
       {/* FIXME: replace the builder page link */}
       <BuilderHeader address={address} name={builderName} builderPageLink="#" className="mt-8" />
-      <div
-        className="w-full mt-6 border border-v3-bg-accent-40 rounded-lg gap-3 flex flex-col divide-y divide-v3-bg-accent-40"
-        data-testid="builderCardContent"
-      >
-        <RewardsInfo {...backerRewardPct} estimatedRewards={estimatedRewards} />
-        {isConnected && (
-          <AllocationInput
-            allocation={allocation}
-            existentAllocation={existentAllocation}
-            maxAllocation={maxAllocation}
-            rifPriceUsd={rifPriceUsd}
-            allocationTxPending={allocationTxPending}
-            onAllocationChange={onAllocationChange}
-            className="px-2 py-3 mx-3"
-          />
-        )}
-        {isConnected && <CurrentBacking existentAllocation={existentAllocation} />}
-      </div>
-      {isConnected && existentAllocation !== 0n && (
-        <Button
-          variant="secondary"
-          className="border-v3-bg-accent-40 px-2 py-1 mt-6"
-          textClassName="text-[14px] font-normal"
-          onClick={() => onAllocationChange(0)}
-          data-testid="removeBackingButton"
-        >
-          Remove backing
-        </Button>
+      {!isRewardable && (
+        <div className="pt-5">
+          <Warning />
+        </div>
```

Comment:
NIT: Please check, but I'm pretty sure this work the same without the extra div. similar thing for line 91

```suggestion
      <Warning className="mt-5" />
```

Reply:
I've had to change the padding top to adhere to the design